backend/src/main/java/ca/bc/gov/app/routes/BaseRouter.java

```diff
@@ -0,0 +1,15 @@
+package ca.bc.gov.app.routes;
+
+import org.springframework.web.reactive.function.server.RouterFunction;
+import org.springframework.web.reactive.function.server.ServerResponse;
+
+public abstract class BaseRouter {
```

Comment:
should it be an interface since no implementation methods?

Reply:
True. It used to have some methods but I moved to another class and forgot to refactor this one